src/index.html

```diff
@@ -1,17 +1,245 @@
     <title>HTML Form</title>
     <link rel="stylesheet" href="./style.css">
   </head>
+
   <body>
-    <h1>HTML Form</h1>
+    <form
+      action="https://mate-academy-form-lesson.herokuapp.com/create-application"
+      method="post"
+      name="personalInformation"
+    >
+      <fieldset class="form-fieldset">
+        <legend>Personal information</legend>
+        <div class="form-field">
+          <label for="surname">
+            Surname:
+
+            <input
+              type="text"
+              autocomplete="off"
+              id="surname"
+              name="surname"
+            >
+
+          </label>
+        </div>
+        <div class="form-field">
+          <label for="name">
+            Name:
+
+            <input
+              type="text"
+              autocomplete="off"
+              id="name"
+              name="name"
+            >
+
+          </label>
+        </div>
+        <div class="form-field">
+          <label for="age">
+            How old are You?
+
+            <input
+              type="number"
+              id="age"
+              min="1"
+              max="100"
+              value="12"
+              name="age"
+            >
+
+          </label>
+        </div>
+        <div class="form-field">
+          <label for="date">
+            Full date of birth:
+
+            <input
+              type="date"
+              id="date"
+              name="date"
+            >
+
+          </label>
+        </div>
+        <div class="form-field">
+          <label for="agreement">
+            I accept the term of the agreement
+
+            <input
+              type="checkbox"
+              name="agreement"
+              id="agreement"
+              required
+            >
+
+          </label>
+        </div>
+      </fieldset>
+      <fieldset class="form-fieldset">
+        <legend>Registration</legend>
+        <div class="form-field">
+          <label for="email">
+            E-mail:
+
+            <input
+              type="email"
+              name="email"
+              id="email"
+              placeholder="email@example.com"
+            >
+
+          </label>
+        </div>
+        <div class="form-field">
+          <label for="password">
+            Password:
+
+            <input
+              type="password"
+              name="password"
+              id="password"
+              minlength="6"
+              maxlength="15"
+            >
+
+          </label>
+        </div>
+      </fieldset>
+      <fieldset class="form-fieldset">
+        <legend>An interesting fact about you!</legend>
+        <div class="form-field">
+          <label for="cats">
+            Do you love cats?
+
+            <input
+              type="radio"
+              name="cats"
+              id="catsYes"
+            >
+            Yes
+
+            <input
+              type="radio"
+              name="cats"
+              id="catsNo"
+            >
+            No
```

Comment:
Make **Yes** and **No** values a clicable lables so it focuses on the correct input when clicked